doc/en/user/source/styling/sld/working.rst

```diff
@@ -35,7 +40,22 @@ using the :ref:`data_webadmin_layers` page of the :ref:`web_admin`.
 .. note:: When adding a layer and a style for it to GeoServer at the same time, the style should be added first, 
           so that the new layer can be associated with the style immediately. 
 
-          
+Style Groups
```

Comment:
Consider rewriting to make this section focus on style layer descriptor, used to define the operation of a wms.

> Style Layer Descriptor Style
> 
> A :ref:`sld_reference_sld` document containing any number of ``<NamedLayer>`` and ``<UserLayer>`` elements,   each of which may contain any number of ``<UserStyle>`` or ``<NamedStyle>`` elements.
>
> Within a style layer descriptor document the name of any ``<NamedLayer>`` elements should match a layer (or layer group) in the catalog. Likewise,  ``<NamedStyle>`` elements should refer to a style in the catalog.
>
> etc...